torch/testing/_internal/common_utils.py

```diff
@@ -173,6 +174,15 @@ def _get_test_report_path():
 parser.add_argument('--import-slow-tests', type=str, nargs='?', const=SLOW_TESTS_FILE)
 parser.add_argument('--import-disabled-tests', type=str, nargs='?', const=DISABLED_TESTS_FILE)
 
+# Only run when -h or --help flag is active to display both unittest and parser help messages.
+def run_unittest_help(argv):
+    unittest.main(argv=argv)
+
+if '-h' in sys.argv or '--help' in sys.argv:
+    help_thread = threading.Thread(target=run_unittest_help, args=(sys.argv,))
```

Comment:
Is creating a new thread necessary for displaying a help message here?

Reply:
When you run a test with -h, it seems like the execution stops at `args, remaining = parser.parse_known_args()` and displays the help message. However, also when unittest is run, it also takes over the thread to display the help message, so I thought to display both help messages from parser and unittest a new thread or subprocess would be needed.